talent/job-descriptions/offers/lead-applications.md

```diff
@@ -0,0 +1,37 @@
+Lead Software Engineer in the Applications team
+------------------------------------------
+
+### Location & Seniority
+
+**Location:** Madrid / Fully remote  
+**Seniority:** Lead 
+
+**Compensation:** 58,000 Euros + [stock options](https://github.com/src-d/guide/blob/master/talent/esop.md)
+
+### [Company](../company-section.md)
+
+### [Engineering at source{d}](../engineering-section.md)
+
+### Role
+
+This position is for a Lead Engineer in the Applications team.
+
+- As a Lead Engineer, you will be expected to plan current and future work, provide mentorship, overseeing progress of the team and, in general, be able to make day-to-day work smooth for everyone in the team.
```

Comment:
overseeing -> oversee
(expected to... oversee)

Or: _provide mentorship overseeing progress of the team_